fail2ban/server/utils.py

```diff
@@ -116,7 +117,22 @@ def setFBlockMode(fhandle, value):
 		return flags
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60, shell=True, output=False, tout_kill_tree=True, success_codes=(0,)):
+	def buildShellCmd(realCmd, varsDict):
```

Comment:
what is the purpose of this function? (seems to be "To append to the command list of environment variables from varsDict" or smth like that?)
also if we take the point of "order of items in dict is not guaranteed" in a wider sense of "order might differ between runs of `.*items`" this wouldn't work -- might better first to sort keys or smth like that to get a deterministic order